amostra/schemas/container.json

```diff
@@ -21,10 +21,13 @@
             "description": "Maps a Sample Foreign key to a location"
         }
     },
+    "additionalProperties": false,
     "required": [
         "uuid",
         "revision",
-        "name"
+        "name",
+        "kind",
```

Comment:
Why does this testing PR have to change what is required?

Reply:
In code here, both kind and content needed.
https://github.com/NSLS-II/amostra/blob/master/amostra/objects.py#L175

In design, 
contents needed make sense, not sure about kind

Reply:
it doesn't have really strong connection with this PR. I updated them to make fake container works(has kind and contents).

Reply:
OK, then let's leave the schema as is. Schema changes should be motivated by user needs, not testing.

Reply:
I don't understand why you marked this as "resolved".